src/cgame/cg_buildable.cpp

```diff
@@ -630,6 +630,7 @@ void CG_InitBuildables()
 		//Prefer md5 models over md3
 
 		if ( cg_highPolyBuildableModels.integer &&
+		     CG_FileExists( va( "models/buildables/%s/%s.iqm", buildableName, buildableName ) ) &&
 		     ( bi->models[ 0 ] = trap_R_RegisterModel( va( "models/buildables/%s/%s.iqm",
```

Comment:
By the way, looking at RE_RegisterModel(), it seems the value 0 is used for errors, so maybe we can add a check to the trap function (or the engine-side function) and return 0 if the file doesn't exist. This will avoid having to check each time.